spec/features/service_drafts/forms/options_form_spec.rb

```diff
@@ -7,7 +7,42 @@
 
   context 'when submitting the form' do
     before do
-      select 'Spatial', from: 'Subset Types'
+      check 'Spatial'
```

Comment:
Is there a reason why the other types were omitted ( line, circle)?

Reply:
I did not feel that every type needed to be populated and saved, but rather each unique type (based on the fields they contain). Circle has the same fields as Point, and Line has the same fields as Polygon, so I felt that those types were covered.